### ISSUE TITLE
EMV features unavailable when passing tokens with deprecated session manager method

### ISSUE THREAD

**Issue by brblakley** [CardFlight/cardflight-ios#46]
Steps to reproduce (using CardFlight iOS SDK 3.2 with iOS 9.3.1):

- Call `[[CFTSessionManager sharedInstance] setLogging:YES]`.
- Call `-[CFTSessionManager setApiToken:accountToken:]` with an EMV-compatible account’s tokens.
- Note that the console doesn’t ever report “is EMV ready:Yes” like it did in 3.1 (but does seem to appear / work correctly when using 3.2’s new `setApiToken:accountToken:completed:`).
- Connect an EMV-compatible reader.
- In `-[CFTReaderDelegate readerIsConnected:withError:]`, call `-[CFTReader beginEMVTransactionWithAmount:andChargeDictionary:]`. 

Expected result: `-[CFTReaderDelegate emvMessage:]` called with `EMVMessage_DIP_OR_SWIPE`.

Actual result: `-[CFTReaderDelegate emvMessage:]` called with `EMVMessage_USE_MAG_READER`. (Dipping a card causes this call to be made repeatedly.)

Log:
```
2016-05-05 15:30:20.021 CardFlight -[CFTSessionManager setApiToken:accountToken:] DEPRECATED, USE setApiToken:accountToken:completed: INSTEAD
2016-05-05 15:30:20.998 CardFlight -[CFTWalkerReader onNoDeviceDetected]
2016-05-05 15:30:40.691 CardFlight -[CFTReader readerAttached] Call to readerIsAttached UNHANDLED
2016-05-05 15:30:40.699 CardFlight -[CFTReader walkerConnected:] Walker Detected
2016-05-05 15:30:40.699 MyReaderDelegate readerIsConnected:YES withError:(null)
2016-05-05 15:30:40.700 CardFlight -[CFTReader walkerConnected:] Call to readerIsConnected:YES withError:nil
2016-05-05 15:30:41.415 CardFlight -[CFTWalkerReader onReturnDeviceInfo:] - {
    batteryLevel = "4.040";
    batteryPercentage = 94;
    bootLoaderVersion = "3.04";
    csn = REDACTED;
    emvKsn = REDACTED;
    firmwareVersion = "8.5.3.60";
    hardwareVersion = "7.4";
    isCharging = 0;
    isSupportedNfc = 0;
    isSupportedTrack1 = 1;
    isSupportedTrack2 = 1;
    isSupportedTrack3 = 0;
    isUsbConnected = 0;
    pinKsn = REDACTED;
    trackKsn = REDACTED;
    uid = REDACTED;
}
2016-05-05 15:30:44.619 CardFlight beingEMVTransactionWithAmount:1
2016-05-05 15:30:44.621 MyReaderDelegate emvMessage:38 (Swipe card)
```

Note that this isn’t an issue for my purposes since I’ll be using the new `setApiToken:accountToken:completed:` method, but I figured I should report it all the same.

**Comment by hokiewalrus** [comment]
Functionality will be restored in the next release

**Comment by hokiewalrus** [comment]
Fixed in version 3.2.1
